### PR TITLE
Fix label overflow

### PR DESCRIPTION
**Pull request by muddana-satish**
Signed-off-by: muddana-satish <satish.muddana@ericsson.com>

### CODE REVIEW

**Comment by muddana-satish** [comment]
I broke this in #69. For some reason, the PIXI BitMap doesn't update the text correctly. Works fine if I create a new object.

**Comment by muddana-satish** [comment]
Before:

image


After:

image

P.S - This issue is noticeable only when the GPU support is off.

**Review by bhufmann — CHANGES_REQUESTED** [review]

File: timeline-chart/src/components/time-graph-row-element.ts [hunk 1]
```diff
@@ -80,10 +81,10 @@ export class TimeGraphRowElement extends TimeGraphComponent {
             }
         }
 
-        textObj.text = displayLabel;
-        textObj.x = textObjX;
-        textObj.y = textObjY;
-        this.displayObject.addChild(textObj);
+        const newTextObj = new PIXI.BitmapText(displayLabel, { fontName: fontName });
```
[the remark this answers is not in the document]
Comment (muddana-satish) [reply timeline-chart/src/components/time-graph-row-element.ts:85]: Yes. Modified it. Thanks for the suggestion 👍

**Review by bhufmann — APPROVED** [review]
Looks good to me

**Review by MatthewKhouzam — APPROVED** [review]

**Pull request merged**